ui/next.config.js

```diff
@@ -15,6 +29,19 @@ const nextConfig = {
                 permanent: true
             }
         ];
+    },
+    async headers() {
```

Comment:
Are you adding other headers beyond CSP? Examples for hedd here - https://github.com/JiscSD/hedd-axolotl/blob/main/axolotl/next.config.js (note frame-src is set to * due to payment page loading iframes from banks)

Reply:
Have included most of the extra headers from the hedd example, only leaving out X-XSS-Protection because MDN said it wasn't needed unless you're really trying to support legacy browsers

Reply:
Great, after I sent you the link I revisited Hedd's and noticed they probably need updating as some are replaced by CSP.